clang/docs/UsersManual.rst

```diff
@@ -2486,6 +2486,23 @@ are listed below.
 
     $ clang -fuse-ld=lld -Oz -Wl,--icf=safe -fcodegen-data-use code.cc
 
+Strict Aliasing
+---------------
+
+Clang by default applies C/C++'s strict aliasing rules during optimizations. In
```

Comment:
I believe this isn't the case for `clang-cl`.

Reply:
Thanks updated.

Reply:
Thanks! It looks good now.